### DISCUSSION TITLE
更改头像源后，出现corb跨域问题

### DISCUSSION THREAD

**Discussion by OrdChaos** [walinejs/waline#903] [Q&A] [answered]
如题，此时无法加载头像，控制台提示

network.ts:100 Cross-Origin Read Blocking (CORB) 已屏蔽 MIME 类型为 text/html 的跨域响应 https://(镜像站).com/?url=https%3A%2F%2Fseccdn.libravatar.org%2Favatar%2Fd61f685fbb292ad7e081e2eafebc040b。如需了解详情，请参阅 https://www.chromestatus.com/feature/5629709824032768

已确认不是镜像的问题，求教

**Comment by lizheming** [comment, marked as answer]
最终地址发一下，应该就是镜像站的问题呀，错误信息说的很清楚，返回的类型是 text/html 啊

**Reply by OrdChaos** [reply]
解决了......应该是输错了。地址后面要加/avatar/，我被GRAVATAR_STR的默认参数迷惑了，以为不用加
